src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -61,6 +61,9 @@ const useSimpleWebsocket = (url, protocol = undefined) => {
     socket.onmessage = (event: MessageEvent) => {
       try {
         const json = JSON.parse(event.data);
+        if (json.message.message_type === 'text') {
+          json.message.attachments = [];
+        }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change introduces a potential data loss issue. If the message type is 'text', any existing attachments are being cleared. This could lead to unexpected behavior if there are valid attachments associated with text messages that need to be preserved. 

If the intention is to clear attachments only when they are not relevant or valid, it would be better to add a validation check for the attachments before clearing them.

```diff
-         if (json.message.message_type === 'text') {
-           json.message.attachments = [];
-         }
+         if (json.message.message_type === 'text' && !isValidAttachments(json.message.attachments)) {
+           json.message.attachments = [];
+         }
```

In this suggested change, `isValidAttachments` is a hypothetical function that you would need to implement according to your specific validation rules.


<!-- This is an auto-generated comment by OSS CodeRabbit -->